File: codes_python/A1-013.py
c = str(input())
pwd = int(input())

if c != 'H' and pwd != 4567:
    print('safe locked')
elif c == 'H' and pwd != 4567:
    print('safe locked - change digit')
elif c != 'H' and pwd == 4567:
    print('safe locked - change char')
elif c == 'H' and pwd == 4567:
    print('safe unlocked')

File: codes_c/A1-013.c
#include <stdio.h>
#include <string.h>
int main()
{
    char c;
    char code[10];
    scanf(" %c", &c);
    scanf("%s", code);
    if (c == 'H' && strcmp(code, "4567") == 0)
        printf("safe unlocked\n");
    else if (c == 'H')
        printf("safe locked - change digit\n");
    else if (strcmp(code, "4567") == 0)
        printf("safe locked - change char\n");
    else
        printf("safe locked\n");
    return 0;
}
